init_floating_base: [0, 0, 0.45, 0, 0.7071, 0, 0.7071]
init_joint_angles: [-1.18542, -1.84009, -1.90918, 1.5428, 1.18639, -2.28165, 1.20091, -1.84009, -1.90918, -1.54403, 1.20184, -0.859485]
target_pose_R:
  position: [0.6624722, -0.2999992, 0.2150011]
  orientation:
    - [-1, 3.403356e-10, 9.265359e-05]
    - [9.265359e-05, 3.673205e-06, 1]
    - [6.776264e-21, 1, -3.673205e-06]
target_pose_L:
  position: [0.5825371, 0.4000081, 0.1799985]
  orientation:
    - [-0.7603741, -0.6494853, 2.385352e-06]
    - [-0.6494853, 0.7603741, -6.466215e-06]
    - [2.385952e-06, -6.465994e-06, -1]